napari/layers/points/points.py

```diff
@@ -1100,11 +1100,11 @@ def _get_state(self):
                 'symbol': self.symbol,
                 'edge_width': self.edge_width,
                 'face_color': self.face_color,
-                'face_color_cycle': self.face_color_cycle,
+                # 'face_color_cycle': self.face_color_cycle,
```

Comment:
why are these commented out?  they are breaking `test_serialize.py` at `# Check number of properties is same as number in signature`

Reply:
oh I see, cause otherwise `test_layer_save` fails at `# Compare layer metadata`...

Reply:
ah yes - this is a problem. Maybe @kevinyamauchi or @jni can help me fix it. We can't return a `Cycle` here as that object doesn't serialize well - instead I want to do something like `list(self.face_color_cycle)` but that goes on for ever. One solution would be to not accept cycles for this property but only accept lists and then call `cycle` on the list elsewhere as a private attribute. Thoughts?

Reply:
> Where n is the length of the desired list, which I guess we have access to here?

I'm not actually sure if we do? I can't recall - maybe @kevinyamauchi can quickly jump in here

Reply:
Just had a good slack with @kevinyamauchi - he is going to make a small standalone PR to convert the `face_color_cycle` and `edge_color_cycle` to a list and change the doc strings/ typing appropriately (both in Points and Vectors) and then add a private property with the actual cycle on it that gets used. Once that PR is merged we will update this one accordingly

Reply:
I have made the PR in #1163 . I need to do a bit more testing, but I think it is good to go.